manual/source/Tutorial/DataStructures/Alignment/ScoringSchemes.rst

```diff
@@ -0,0 +1,187 @@
+To do so, we simply call the function :dox:`globalAlignment` and give as input parameters the ``align`` object and the Blosum62 scoring scheme.
+The globalAlignment function returns the score of the best alignment, which we store in the ``score`` variable that is then printed together with the corresponding alignment.
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp
+   :fragment: alignment
+
+The output of a global alignment implementing the Blosum62 scoring function is as follows:
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp.stdout
+
+.. note::
+   As can be noted the output of this scoring scheme is completely different with respect to the output generated with the simple scoring scheme confirming that the scoring scheme choice is one of the most important step to achieve high quality alignments.
+
+Gap Evaluation
+--------------
+
+In the previous sections we proposed two simple code examples useful to highlight the differences between two scoring schemes capable to evaluate match and mismatch events. In this section we will see the three gap models, implemented in the SeqAn library, to evaluate the insertion and deletion events.
+
+Linear Gap Model
+^^^^^^^^^^^^^^^^
+
+The easiest is the Linear gap model that considers, for the alignment score computation, the gap length (g) giving the possibility to evaluate with different scores gaps of different sizes;
+
+.. image:: linear.png
+   :width: 160px
+   :align: center
+
+This gap model is chosen as standard when only a gap value is provided in the scoring function or when the two provided gaps have the same value. For instance, this gap model as been adopted during the alignment computation of the two proposed examples.
+
+Affine Gap Model
+^^^^^^^^^^^^^^^^
+
+It has been proven that the first amino acid or nucleotide inserted/deleted (identified as gap open) found during the alignment operations is more significant, from a biological point of view, than the subsequent ones (called gap extension), making the so called Affine Gap model a viable solution for the alignment of biomolecules.
+Affine gap model that attribute different costs to the gap open (d) and the gap extension (e) events, is able to assign an higher penalty to the gap presence with respect to its relative length (g).
+
+.. image:: affine.png
+   :width: 240px
+   :align: center
+
+The Affine Gap model implemented in the DP alignment algorithms is however quite expensive both in terms of computational time as well as in terms of memory requirements with respect to other less demanding solutions such as the Linear Gap model application.
+
+Dynamic Gap Model
+^^^^^^^^^^^^^^^^^
+
+An optimised version of the Affine Gap model called Dynamic Gap Selector (DGS) designed by Urgese et al. :cite:`Urgese2014`. This new gap model can be used to reduce the computational time and the memory requirement while keeping the alignment scores close to those computed with the Affine Gap model.
```

Comment:
There are two main motivations for keep "AffineGap" as default mode:
1) The DGS method is unknown in the user community while the Affine is widely known;
2) In order to assess the DGS reliability in the score computation should be executed more deeper tests in several alignment tasks.

Concerning the second question, if both query and subject strings gets longer, the difference could increased as well since the alignment matrix became bigger and different alignment paths can be chosen during the alignment procedure.